src/options.rs

```diff
@@ -113,6 +115,11 @@ pub struct BuildOptions {
     #[structopt(long = "target-dir")]
     /// Target dir option to pass to cargo build.
     pub target_dir: Option<String>,
+
+    #[structopt(skip = false)]
```

Comment:
I think the `skip = false` bit isn't necessary, is it? By default structopt will not skip fields.

Or is this trying to make this option skipped, so that it is only exposed with `cargo fuzz coverage`?

Reply:
> Or is this trying to make this option skipped, so that it is only exposed with cargo fuzz coverage?

Yes, the syntax here is `skip = <default value>`, which says that the option should not be shown to the user and specifies a default value. In our case the default value is `false`. I agree that this looks confusing.

My intention was to avoid exposing this flag to the user. It's a build flag that should always be used with the `coverage` command, and the code ensures that this flag is set with that command.

Reply:
Ah okay, thanks for the explanation. Would you mind explaining this a bit more in the comment? Thanks.